pkg/urbit/noun/nock.c

```diff
@@ -1769,6 +1951,17 @@ _n_hint_fore(u3_cell hin, u3_noun bus, u3_noun* clu)
       *clu = u3_nul;
     } break;
 
+    case c3__xray : {
+      u3_noun pri, tan;
+      if ( c3y == u3r_cell(*clu, &pri, &tan) ) {
+        c3_l pri_l = c3y == u3a_is_cat(pri) ? pri : 0;
+        u3t_slog_cap(pri_l, u3i_string("bytecode of"), u3k(tan));
+        _xray(pri_l, fol);
```

Comment:
This prints "bytecode" twice. The caption here should just be `u3k(tan)`.

Reply:
rewrote so that _slog_bytecode just slogs, no slog with caption, thus the two call sites will render as expected and either render one caption or none at all.